docs/section19/19-7.md

```diff
@@ -0,0 +1,84 @@
+        - Case 2: Else if the modeled baseline minimum OA cfm are greater than the modeled proposed minimum OA cfm and a zone served by the HVAC in the baseline has demand controlled ventilation modeled in the proposed (Exception 1) then the outcome is Fail: `elif modeled_baseline_total_zone_min_OA_CFM > modeled_proposed_total_zone_min_OA_CFM and was_DCV_modeled_baseline == false and was_DCV_modeled_proposed == true: outome = "FAIL" and raise_message "For <insert hvac.id> the baseline modeled minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. It appears that demand-controlled ventilation was modeled in the proposed and not the baseline model and that demand-controlled ventilation may be double accounted for in the model (per the HVAC controls and via reduced OA CFM rates in the proposed)."`  
+        - Case 3: Else if the modeled baseline minimum OA cfm are greater than the modeled proposed minimum OA cfm and a zone served by the HVAC in the baseline has an air distribution effectivenss of greater than 1 in the proposed (Exception 2) then the outcome is UNDETERMINED: `elif modeled_baseline_total_zone_min_OA_CFM > modeled_proposed_total_zone_min_OA_CFM and zone_air_distribution_effectiveness_greater_than_1 == true: outome = "UNDETERMINED" and raise_message "For <insert hvac.id> the modeled baseline minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. It appears as though G3.1.2.5 Exception 2 may be applicable. A manual check for this exception is recommended otherwise fail."`  
+        - Case 4: Else if the modeled baseline minimum OA cfm are less than the modeled proposed minimum OA cfm the outcome is FAIL: `elif modeled_baseline_total_zone_min_OA_CFM < modeled_proposed_total_zone_min_OA_CFM: outome = "FAIL" and raise_message "For <insert hvac.id> the modeled minimum ventilation system outdoor air intake flow CFM appears lower than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. Fail unless G3.1.2.5 Exception 3 is applicable. This exception states that where the minimum outdoor air intake flow in the proposed design is provided in excess of the amount required by the building code or the rating authority, the baseline building design shall be modeled to reflect the greater of that required by either the rating authority or the building code and will be less than the proposed design. "`  
+        - Case 5: Else, fail: `Else: outcome = "Fail" and raise_message "For <insert hvac.id> the modeled baseline minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design which does not meet the requirements of Section G3.1.2.5."`  
```

Comment:
I would not use "appear" in the notes because we have established it to be higher, right? But no strong feelings.

Reply:
removed